site/content/user-guide/amqp-adapter.md

```diff
@@ -33,12 +37,21 @@ There is a subtle difference between the *device identifier* (*device-id*) and t
 {{% /note %}}
 
 ### SASL EXTERNAL Authentication
-TODO
+When a device uses a client certificate for authentication using SASL EXTERNAL, the AMQP adapter internally performs the following steps:
+
+* Adapter extracts the client certificate's Issuer DN and uses it to
```

Comment:
IMHO it is worth mentioning that the TLS handshake will succeed if the certificate's validity period includes the current point in time but its signature is not checked (as part of the TLS handshake).

Reply:
i think mentioning this assumes that the adapter is not configured to use a custom trust store, right? Checking the cert's validity happens by default if no trust options is configured for the adapter.

Reply:
that's a good point. Then maybe mention that this is the *default behavior* if no trust anchor is specified explicitly